smart-contracts/contracts-common/concordium-contracts-common/src/types.rs

```diff
@@ -1062,6 +1064,11 @@ impl<'de> SerdeDeserialize<'de> for Duration {
     }
 }
 
+/// Canonical address of an account, as raw bytes.
+#[derive(Eq, PartialEq, Copy, Clone, PartialOrd, Ord, Debug, Hash)]
+#[cfg_attr(feature = "fuzz", derive(Arbitrary))]
```

Comment:
I don't think we need to derive `Arbitrary`, the `fuzz` feature is not really meant for external use and not really maintained.
```suggestion
```